src/cloudsc_cuda/cloudsc/load_state.cu

```diff
@@ -0,0 +1,451 @@
+/*
```

Comment:
That shows my ignorance for CUDA C: Is it customary to have all host-side code also with the ending `.cu`? I would have expected the various `cloudsc_driver`, as well as `cloudsc_validate`, `load_state` should actually be `.c` or `.cpp` files?